Automattic/wc-calypso-bridge
# Track events inside OBW and checklist

Fixes #85 

This PR adds event tracking for the following events:
* OBW step completion
* Tasklist item clicked
* Tasklist finished (clicked "I'm done")
* Edit address in OBW clicked
* Calypsoify manually turned on/off

### Screenshots
<img width="809" alt="screen shot 2018-11-01 at 3 41 10 pm" src="https://user-images.githubusercontent.com/10561050/47878557-995f9a80-ddec-11e8-822e-0ff419a33514.png">

### Testing
1.  Make sure you have a Jetpack connected site with tracking enabled.
2.  Try the various above events to make sure they come through (may take a few minutes to show up) - https://mc.a8c.com/tracks/live/?eventname=%25atomic_%25&user=&useragent=

### Notes
* The javascript events are prefixed automatically with `jetpack_`
* The task list completion event is dependent on https://github.com/Automattic/wc-calypso-bridge/pull/130
